proposals/nnnn-add-supported-rgb-colors.md

```diff
@@ -0,0 +1,65 @@
+# Add Supported RGB Colors to Light Capabilities
+
+* Proposal: [SDL-NNNN](nnnn-add-supported-rgb-colors.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
```

Comment:
```suggestion
* Impacted Platforms: [Core / iOS / Java Suite / RPC]
```

We are now using `Java Suite` to encompass libraries for Android, Java SE, and Java EE.

Reply:
updated, thanks